components/body/index.styl

```diff
@@ -0,0 +1,86 @@
+.main-content-container
+  padding-top 20px
+
+.main-side-margin
+  margin 0 20px
+
+.section-title
+  border-top 1px solid #CCC
```

Comment:
Might be worth pulling `#CCC` out into a variable like `gray-light-color`. In Force/MG we extract a [stylus_lib](https://github.com/artsy/force/blob/master/components/stylus_lib/index.styl) component that holds just a bunch of variables and mixins, then use it in stylesheets with `@import '../stylus_lib' at the top.